src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -368,6 +370,13 @@ def _replace_weights(self, dense_weight_generator, model):
             module = operator.attrgetter(prefix)(model)
             update_parameter_data(module, data, param_name)
 
+    def _find_sparse_compression_targets(self, model: Module) -> Set[str]:
```

Comment:
This seems like an unnecessary function and find_compression_argets can be called in place passing in the sparsity_config

Reply:
Removed!